frontend/ui/src/pages/Inbox/MessageInput/index.tsx

```diff
@@ -49,116 +54,126 @@ interface SelectedSuggestedReply {
   message: SuggestedReply;
 }
 
-const contentResizedHeight = 200;
-
 const MessageInput = (props: Props) => {
-  const {source, conversation, suggestions, showSuggestedReplies, hideSuggestedReplies, sendMessages} = props;
+  const {
+    source,
+    conversation,
+    suggestions,
+    showSuggestedReplies,
+    hideSuggestedReplies,
+    sendMessages,
+    draggedAndDroppedFile,
+    setDraggedAndDroppedFile,
+  } = props;
+
+  const contentResizedHeight = 200;
 
   const outboundMapper = getOutboundMapper(source);
+  const fileOutboundMapper = getOutboundMapper('facebook') as FacebookMapper;
   const channelConnected = conversation.channel.connected;
```

Comment:
Here I'm casing the `outboundMapper `for files as the `Facebook` one, as we don't support sending attachments for others sources (obviously this has to be removed when we do)